1_The_Physical/Guess_Point_Level_1_Stateless/1_Red_Green_Refactor/1_1_Fizzbuzz/src/index.spec.ts

```diff
@@ -1,5 +1,41 @@
-
+import fizzbuzz from './fizzbuzz';
 describe("fizzbuzz", () => {
 
+  it("return error if number is less than 1", () => {
+    expect(() => fizzbuzz(0)).toThrow();
+  });
+
+  it("return error if number is more than 100", () => {
+    expect(() => fizzbuzz(101)).toThrow();
+  });
+
+  it("returns a string", () => {
+    expect(typeof fizzbuzz(5)).toBe('string');
+  });
+
+  it("returns the number as a string", () => {
+    expect(fizzbuzz(1)).toBe('1');
+  });
+
+  it("returns Fizz for multiples of 3 and not multiples of 5", () => {
+    for(let i = 3; i < 100; i += 3) {
+      if(i % 5 === 0) continue;
```

Comment:
An alternative one-line could be:
```
if(i % 5 !== 0) expect(fizzbuzz(i)).toBe('Fizz');
```
with the same idea for the multiples of 5 test. However, negation expressions can be less intutive, so either way it's all good 👍

Reply:
I believe that negation expressions can be less intuitive. So I will stick with that on this case.